From 48a886d7594ec54d036b1b591fabf3d89a33f7a8 Mon Sep 17 00:00:00 2001
From: Kasparas <kasparas6@hotmail.com>
Date: Fri, 16 Jun 2017 04:50:44 +0300
Subject: [PATCH] Don't abort even if setexeccon failed

---
 Utils.cpp | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Utils.cpp b/Utils.cpp
index 6ffdd7f..abe2b19 100644
--- a/Utils.cpp
+++ b/Utils.cpp
@@ -284,12 +284,12 @@ status_t ForkExecvp(const std::vector<std::string>& args, security_context_t con
 
     if (setexeccon(context)) {
         LOG(ERROR) << "Failed to setexeccon";
-        abort();
+        //abort();
     }
     status_t res = android_fork_execvp(argc, argv, NULL, false, true);
     if (setexeccon(nullptr)) {
         LOG(ERROR) << "Failed to setexeccon";
-        abort();
+        //abort();
     }
 
     free(argv);
@@ -316,12 +316,12 @@ status_t ForkExecvp(const std::vector<std::string>& args,
 
     if (setexeccon(context)) {
         LOG(ERROR) << "Failed to setexeccon";
-        abort();
+        //abort();
     }
     FILE* fp = popen(cmd.c_str(), "r");
     if (setexeccon(nullptr)) {
         LOG(ERROR) << "Failed to setexeccon";
-        abort();
+        //abort();
     }
 
     if (!fp) {
-- 
2.7.4

